packages/core/src/diff/node.rs

```diff
@@ -379,8 +382,8 @@ impl VNode {
                     } else {
                         to.remove_node(id);
                     }
-                    dom.reclaim(id)
                 }
+                dom.reclaim(id)
```

Comment:
Same thing here. If the node was created under suspense [here](https://github.com/DioxusLabs/dioxus/blob/797e7492a1b9e188627ab7530a66607ba19376d6/packages/core/src/diff/node.rs#L672-L674). I don't think it would have an id